enterprise/backend/test/metabase_enterprise/sandbox/query_processor/middleware/row_level_restrictions_test.clj

```diff
@@ -250,13 +250,6 @@
   (mt/test-drivers (into #{}
                          (filter #(isa? driver/hierarchy % :sql))
                          (mt/normal-drivers-with-feature :nested-queries))
-    (testing "When querying with full permissions, no changes should be made"
```

Comment:
Removed this because it no longer reflects expected behavior. Permissions for a group don't override a sandbox in the same group; only in other groups.